### PR TITLE
Add July conda and conda-build release announcement blog post

### PR DESCRIPTION
**Pull request by beeankha**
xrefs https://github.com/conda/conda-build/issues/4926 and https://github.com/conda/conda/issues/12849

### CODE REVIEW

File: blog/2023-07-28-july-releases.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+title: "July Releases for Conda and Conda-Build"
```
Comment (tnabtaf) [inline blog/2023-07-28-july-releases.md:2]: Think about saying "July 2023 Releases".

File: blog/2023-07-28-july-releases.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+## [Changes in Conda-Build](https://github.com/conda/conda-build/releases/tag/3.26.0)
+
+To update conda-build, run:
+```bash
+conda install -n base conda-build=3.26.0
+```
+
+### ✨ What's New? ✨
+
+In the latest version of conda-build, extra-meta data is logged to make it easier to verify that the right extra-meta data is burned into packages (which also helps to co-relate packages and their build-log). The feature was first introduced in [PR #4303](https://github.com/conda/conda-build/pull/4303) and is now improved via the logging call.
```
Comment (tnabtaf) [inline blog/2023-07-28-july-releases.md:85]: My brain rebelled against the syntax of "extra-meta data."  Would there be less cognitive dissonance it that is rendered as "`extra-meta` data" instead?

File: blog/2023-07-28-july-releases.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+* [Executable invocations (e.g., `conda-build`)](https://github.com/conda/conda-build/pull/4921)
+* [Module-based invocations (e.g., `python -m conda_build.cli.main_build`)](https://github.com/conda/conda-build/pull/4921)
+
+
+* * * 
+
+### We ❤️ Our Community
+
+Altogether we had 10 (!) new contributors to the conda and conda-build repositories this release cycle; thank you to all of our open source community members for helping make the new versions of conda and conda-build so great.
```
Comment (tnabtaf) [inline blog/2023-07-28-july-releases.md:122]: I know it's tedious, but would it be worth the effort add a list of new contributors, and links to their GitHub pages?

I can submit a PR if the this is worthwhile, but the thought of creating it makes you shake.

**Review by tnabtaf — APPROVED** [review]
Everything is a suggestion.

File: blog/2023-07-28-july-releases.md (hunk 1 above)
Comment (jezdez) [inline blog/2023-07-28-july-releases.md:2]:
```suggestion
title: "July 2023 Releases for conda and conda-build"
```

File: blog/2023-07-28-july-releases.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+slug: "2023-07-28-july-2023-releases"
+description: |
+    Conda 23.7.0 and conda-build 3.26.0 have been released! 🎉
+authors: [beeankha]
+tags: [announcement, conda, conda-build]
+image: img/blog/2023-07-28-july-releases/rocket.jpg
+
+---
+
+July releases are here! 🎊 Conda 23.7.0 and conda-build 3.26.0 are now available on both `main` and `conda-forge` channels.
```
Comment (jezdez) [inline blog/2023-07-28-july-releases.md:12]:
```suggestion
The July 2023 releases are here! 🎊 Conda 23.7.0 and conda-build 3.26.0 are now available on both `main` and `conda-forge` channels.
```

File: blog/2023-07-28-july-releases.md (hunk 2 above)
Comment (jezdez) [inline blog/2023-07-28-july-releases.md:85]:
```suggestion
In the latest version of conda-build, `extra-meta` data is logged to make it easier to verify that the right extra-meta data is burned into packages (which also helps to co-relate packages and their build-log). The feature was first introduced in [PR #4303](https://github.com/conda/conda-build/pull/4303) and is now improved via the logging call.
```

File: blog/2023-07-28-july-releases.md (hunk 3 above)
Comment (jezdez) [reply blog/2023-07-28-july-releases.md:122]: @tnabtaf Just add the suggestion here, don't create another PR, this gets harder to review that way

CleanShot 2023-07-27 at 11 27 06@2x

Comment (kenodegard) [reply blog/2023-07-28-july-releases.md:122]:
```suggestion
Altogether we had 10 (!) new contributors to the conda and conda-build repositories this release cycle; thank you to all of our open source community members for helping make the new versions of conda and conda-build so great.

* [@johanneskoester](https://github.com/johanneskoester) made their first contribution in [conda#12683](https://github.com/conda/conda/pull/12683)
* [@josegonzalez](https://github.com/josegonzalez) made their first contribution in [conda-build#3998](https://github.com/conda/conda-build/pull/3998)
* [@kalawac](https://github.com/kalawac) made their first contribution in [conda#12738](https://github.com/conda/conda/pull/12738)
* [@lesteve](https://github.com/lesteve) made their first contribution in [conda#12715](https://github.com/conda/conda/pull/12715)
* [@marcoesters](https://github.com/marcoesters) made their first contribution in [conda#12863](https://github.com/conda/conda/pull/12863)
* [@mattkram](https://github.com/mattkram) made their first contribution in [conda#12730](https://github.com/conda/conda/pull/12730)
* [@morremeyer](https://github.com/morremeyer) made their first contribution in [conda#12871](https://github.com/conda/conda/pull/12871)
* [@mpotane](https://github.com/mpotane) made their first contribution in [conda#11740](https://github.com/conda/conda/pull/11740)
* [@rfezzani](https://github.com/rfezzani) made their first contribution in [conda-build#4850](https://github.com/conda/conda-build/pull/4850)
* [@schackartk](https://github.com/schackartk) made their first contribution in [conda#12781](https://github.com/conda/conda/pull/12781)
```

File: blog/2023-07-28-july-releases.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+* `pip` has been added to the `env-doc make` command so that the function works correctly (`pip` is no longer added by default with the python conda package).
+* Subcommands are implemented as conda plugins.
+* Duplicate `get_summary` calls are dropped in `conda_build.skeletons.pypi`.
+* `resolved_packages` tests that failed due to recent OpenSSL 3.0.8 release to defaults is now fixed.
+
+
+### 🔧 What Got Fixed? 🔧
+
+The following bug fixes were implemented in the 3.26.0 version of conda-build:
```
Comment (kenodegard) [inline blog/2023-07-28-july-releases.md:97]:
```suggestion
The following bug fixes were implemented in conda-build 3.26.0:
```

File: blog/2023-07-28-july-releases.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+### 🔧 What Got Fixed? 🔧
+
+The following bug fixes were implemented in conda 23.7.0:
+
+* `requests.exceptions.JSONDecodeError` is utilized for ensuring compatibility with different `json` implementations used by requests. This fixes a bug that caused only the first of multiple given source URLs to be tried. This also raises the minimum required requests version to 2.27.0.
+* `__osx` virtual package are no longer exported when `CONDA_OVERRIDE_OSX` is set to an empty string.
+* Erroneous `conda deactivate` behavior that unset pre-existing environment variables that are identical to those set during `conda activate` has been fixed.
+* Third-party subcommands now correctly receive _remaining_ arguments instead of a blanket `sys.argv[2:]`, which broke `conda_cli` testing.
+The following bug fix was implemented in conda 23.7.1:
```
Comment (kenodegard) [inline blog/2023-07-28-july-releases.md:50]:
```suggestion

The following bug fix was implemented in conda 23.7.1:
```

File: blog/2023-07-28-july-releases.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
+---
+title: "July 2023 Releases for conda and conda-build"
+slug: "2023-07-28-july-2023-releases"
+description: |
+    Conda 23.7.0 & 23.7.1 and conda-build 3.26.0 have been released! 🎉
```
Comment (kenodegard) [inline blog/2023-07-28-july-releases.md:5]:
```suggestion
    Conda 23.7.0, 23.7.1, 23.7.2, and conda-build 3.26.0 have been released! 🎉
```

File: blog/2023-07-28-july-releases.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+authors: [beeankha]
+tags: [announcement, conda, conda-build]
+image: img/blog/2023-07-28-july-releases/rocket.jpg
+
+---
+
+The July 2023 releases are here! 🎊 Conda 23.7.0 & 23.7.1 and conda-build 3.26.0 are now available on both `main` and `conda-forge` channels.
```
Comment (kenodegard) [inline blog/2023-07-28-july-releases.md:12]:
```suggestion
The July 2023 releases are here! 🎊 Conda 23.7.0, 23.7.1, 23.7.2, and conda-build 3.26.0 are now available on both `main` and `conda-forge` channels.
```

File: blog/2023-07-28-july-releases.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+
+## Changes in Conda [23.7.0](https://github.com/conda/conda/releases/tag/23.7.0) & [23.7.1](https://github.com/conda/conda/releases/tag/23.7.1)
```
Comment (kenodegard) [inline blog/2023-07-28-july-releases.md:15]:
```suggestion
## Changes in Conda [23.7.0](https://github.com/conda/conda/releases/tag/23.7.0)/[23.7.1](https://github.com/conda/conda/releases/tag/23.7.1)/[23.7.2](https://github.com/conda/conda/releases/tag/23.7.2)
```

File: blog/2023-07-28-july-releases.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+
+To update conda, run:
+```bash
+conda install -n base conda=23.7.1
```
Comment (kenodegard) [inline blog/2023-07-28-july-releases.md:19]:
```suggestion
conda install -n base conda=23.7.2
```

File: blog/2023-07-28-july-releases.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+### 🔧 What Got Fixed? 🔧
+
+The following bug fixes were implemented in conda 23.7.0:
+
+* `requests.exceptions.JSONDecodeError` is utilized for ensuring compatibility with different `json` implementations used by requests. This fixes a bug that caused only the first of multiple given source URLs to be tried. This also raises the minimum required requests version to 2.27.0.
+* `__osx` virtual package are no longer exported when `CONDA_OVERRIDE_OSX` is set to an empty string.
+* Erroneous `conda deactivate` behavior that unset pre-existing environment variables that are identical to those set during `conda activate` has been fixed.
+* Third-party subcommands now correctly receive _remaining_ arguments instead of a blanket `sys.argv[2:]`, which broke `conda_cli` testing.
+
+The following bug fix was implemented in conda 23.7.1:
```
Comment (kenodegard) [inline blog/2023-07-28-july-releases.md:51]:
```suggestion
The following bug fix was implemented in conda 23.7.1/23.7.2:
```

**Review by travishathaway — APPROVED** [review]

**Pull request merged**